CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## Master (Unreleased)
 
+- Add `rubocop-capybara` version constraint to prevent sudden cop enabling when it hits 3.0. ([@pirj])
+
 ## 2.18.0 (2023-01-16)
 
-- Extract Capybara cops to a separate repository. ([@pirj])
+- Extract Capybara cops to a separate repository. The `rubocop-capybara` repository is a dependency of `rubocop-rspec` and the Capybara cops are aliased (`RSpec/Capybara/Foo` == `Capybara/Foo`) until v3.0 is released, so the change will be invisible to users until then. ([@pirj])
```

Comment:
IMHO, Would it help users more to have a link to RuboCop Capybara in this changelog?